packages/lab/src/Flow/Sidebar/Sidebar.tsx

```diff
@@ -190,40 +173,35 @@ export const HvFlowSidebar = ({
             onChange={handleDebouncedSearch}
             inputProps={{ autoComplete: "off" }}
           />
-          {nodeGroups ? (
-            <ul id={groupsElementId} className={classes.groupsContainer}>
-              {Object.entries(groups).map((obj) => {
-                return (
-                  <HvFlowSidebarGroup
-                    key={obj[0]}
-                    id={obj[0]}
-                    expandButtonProps={{
-                      "aria-label": labels?.expandGroupButtonAriaLabel,
-                    }}
-                    itemProps={{
-                      "aria-roledescription": labels?.itemAriaRoleDescription,
-                    }}
-                    {...obj[1]}
+          <ul id={groupsElementId} className={classes.groupsContainer}>
+            {Object.entries(groups).map((obj) => {
+              if (flatten) {
+                return obj[1].items?.map((item) => (
+                  <HvFlowDraggableSidebarGroupItem
+                    key={item.label}
```

Comment:
Should we add an `id` prop to the items to avoid using the label as `id` and `key`? Maybe we make the items as `Record<string, HvFlowGroupItem>` instead of `HvFlowGroupItem[]` for simplicity and to be sure that the `id`'s are not repeated.

I'm also suggesting this change because we could add a prop to the `HvFlowNode` like `itemId` to get the `subtitle` directly from the node group's `items` (instead of providing it twice: once in the node groups definition and another time in the node).